docs/src/docs/asciidoc/user/caching-concepts.adoc

```diff
@@ -39,22 +39,23 @@ and technical decision based upon the requirements and assumptions of your appli
 
 You can configure Ehcache to use various data storage areas.
 When a cache is configured to use more than one storage area, those areas are arranged and managed as `tiers`.
-They are organized in a hierarchy, with the lowest tier being called the `authority` tier and the others being part of the `caching` tier.
+They are organized in a hierarchy, with the lowest tier (farther) being called the `authority` tier and the others being part of the `caching` tier (nearer, also called `near cache`) .
 The caching tier can itself be composed of more than one storage area.
 The _hottest_ data is kept in the caching tier, which is typically less abundant but faster than the authority tier.
 All the data is kept in the authority tier, which is slower but more abundant.
 
 Data stores supported by Ehcache include:
 
 * On-Heap Store - Utilizes Java's on-heap RAM memory to store cache entries. This tier utilizes the same heap memory as
-your Java application, all of which must be scanned by the JVM's garbage collector.  The more heap space your JVM
-utilizes, the more your application's performance will be impacted by garbage collection pauses.  This store is
+your Java application, all of which must be scanned by the JVM garbage collector.  The more heap space your JVM
```

Comment:
It was like this before. It don't want the line to look more modified than needed and be hard to compare. I've only removed the "'s"